app/create_app.py

```diff
@@ -10,11 +11,14 @@
 def create_app() -> Flask:
     flask_app = Flask(__name__, static_url_path="/assets")
 
-    csrf = SeaSurf()
-    csrf.init_app(flask_app)
-
     flask_app.config.from_pyfile("config.py")
 
+    # Auto build the static assets if we are in the dev environment.
+    # In production we will most likely install static assets differently.
+    if flask_app.config.get("ENV") == "development":
+
+        build_assets()
+
```

Comment:
This looks like it will run on every request. Perhaps should go somewhere else..?

Reply:
Any suggestions for this? Would be nice to have very little dev work required.